flow-typed/react-intl_vx.x.x.js

```diff
@@ -328,8 +323,14 @@ declare module 'react-intl' {
   >(
     WrappedComponent: React$ComponentType<P>,
     options?: Opts<IntlPropName, true>,
-  ): React_2.ForwardRefExoticComponent<
-    React_2.PropsWithoutRef<WithIntlProps<React_2.PropsWithChildren<P>>> & React_2.RefAttributes<T>, > & {
+  ): React$AbstractComponent<
+    {|
+      ...WithIntlProps<P>,
+      children?: React$Node,
+      ref?: React$Ref<T>,
+    |},
+    mixed,
+  > & {
```

Comment:
This looks like it reflects another missing feature (not to say it'd necessarily be an easy one to build…) in Flowgen: the TS type here uses `&`, presumably reflecting inexact object types on the TS side, but we have exact object types here and so they really need to be spreads.